clcache.py

```diff
@@ -116,6 +116,10 @@ def normalizeBaseDir(baseDir):
         return None
 
 
+def atomicRename(tempFile, destFile):
+    os.replace(tempFile, destFile)
```

Comment:
I'm not sure this function pulls its own weight; `atomicRename` is even longer than `os.replace` and chances are that the latter is already known to readers of the code.

Reply:
ok, replaced by the context manager and os.replace(...)